p4_fuzzer/oracle_util_test.cc

```diff
@@ -10,7 +10,7 @@
 #include "gtest/gtest.h"
 #include "gutil/status.h"
 #include "net/google::protobuf/contrib/parse_proto/parse_text_proto.h"
-#include "platforms/networking/orion/core/sfe/sfe.pb.h"
```

Comment:
This will not compile, can you check if this file is included in the BUILD file ?

Reply:
This file is not defined in BUILD.bazel file.

Reply:
I see the file is included in the BUILD file in pins_infra, why isn't not included in sonic-pins ?

Reply:
For this file only initial porting is upstreamed in sonic-pins. In Pins-infra also this file was not defined in the BUILD file in the initial porting. hence it is missing.